app/labor/email_logic.rb

```diff
@@ -62,10 +69,11 @@ def get_days_until_next_email
   def get_open_rate
     past_sent_emails = @user.email_messages.where(mailer: "DigestMailer#digest_email").limit(10)
 
+    past_sent_emails_count = past_sent_emails.count
+
     # Will stick with 50% open rate if @user has no/not-enough email digest history
-    return 0.5 if past_sent_emails.length < 10
+    return 0.5 if past_sent_emails_count < 10
```

Comment:
`.length` implicitly loads the items in memory but we only need the counts, not the items

Reply:
Removes line 68 as well :+1:

Reply:
@jacobherrington I don't think I understood your comment, is there anything I should remove?

Reply:
No, I was saying this change was good because it removed two lines. 🙃